tests/src/utility/support.rs

```diff
@@ -264,3 +264,7 @@ pub(crate) fn get_stored_value_from_global_state<T: CLTyped + FromBytes>(
         .clone()
         .into_t::<T>()
 }
+
+pub(crate) fn get_receipt_name(nft_receipt: String, page_table_entry: u64) -> String {
+    format!("{}-m-{}-p-{}", nft_receipt, PAGE_SIZE, page_table_entry)
+}
```

Comment:
is nft_receipt ever anything other than:

```
   let receipt_name = utils::get_stored_value_with_user_errors::<String>(
        RECEIPT_NAME,
        NFTCoreError::MissingReceiptName,
        NFTCoreError::InvalidReceiptName,
    );
```
?

If no, maybe move that part into the utility and only pass in the `p` value